app/src/main/java/com/g11x/checklistapp/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.g11x.checklistapp;
+
+/**
+ * Created by wcalabrese on 8/15/16.
+ */
+
+public class Constants {
+    public static final String LOCAL_REPOSITORY_AUTHORITY = "com.g11x.checklistapp.provider";
```

Comment:
🔥  mind blown 🔥 

1. Can this be added to the top level in build.gradle?
2. Please disregard latest change that did it wrong.